combined-MoviePlaylist-1.0.0-resolved.json

```diff
@@ -14,269 +14,105 @@
-      "description": "SwaggerHub API Auto Mocking"
+      "url": "http://localhost:8080",
+      "description": "Inferred Url"
     }
   ],
   "tags": [
     {
-      "name": "Watchlist",
-      "description": "APIs that revolve around the watchlist"
+      "name": "auth-api-controller",
+      "description": "Auth Api Controller"
     },
     {
-      "name": "Authentication",
-      "description": "APIs for logging in and out of the API"
+      "name": "media-api-controller",
+      "description": "Media Api Controller"
     },
     {
-      "name": "Users",
-      "description": "APIs for creating new users"
+      "name": "users-api-controller",
+      "description": "Users Api Controller"
     },
     {
-      "name": "Media",
-      "description": "APIs for searching media"
+      "name": "watchlist-api-controller",
+      "description": "Watchlist Api Controller"
     }
   ],
   "paths": {
-    "/watchlist": {
-      "post": {
-        "tags": [
-          "Watchlist"
-        ],
-        "summary": "Creates a new Watchlist for the calling user",
-        "description": "Creates a new Watchlist for the calling user\n",
-        "responses": {
-          "201": {
-            "description": "item created",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Watchlist"
-                }
-              },
-              "application/xml": {
-                "schema": {
-                  "$ref": "#/components/schemas/Watchlist"
-                }
-              }
-            }
-          }
-        }
-      }
-    },
-    "/watchlist/{watchlist_id}": {
-      "get": {
-        "tags": [
-          "Watchlist"
-        ],
-        "summary": "Retrieve Watchlist",
-        "description": "Gets an existing watchlist for the calling user if watchlist exists and the user has permissions to view the Watchlist.\n",
-        "parameters": [
-          {
-            "name": "watchlist_id",
-            "in": "path",
-            "description": "The id of the watchlist to retrieve",
-            "required": true,
-            "style": "simple",
-            "explode": false,
-            "schema": {
-              "type": "integer"
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Item retrieved",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Watchlist"
-                }
-              },
-              "application/xml": {
-                "schema": {
-                  "$ref": "#/components/schemas/Watchlist"
-                }
-              }
-            }
-          },
-          "404": {
-            "description": "Watchlist not found for user"
-          }
-        }
-      }
-    },
-    "/watchlist/{watchlist_id}/media": {
+    "/MovieWatchlistAPI/1.0.0/auth/register": {
```

Comment:
Hmm so I auto-generated this updated swagger document using SpringDoc since I made a changes to the APIs I was working on; I'm not sure how to get rid of it